spec/scrabble_spec.rb

```diff
@@ -0,0 +1,95 @@
+require './lib/scrabble'
+
+describe Scrabble::Scrabble do
+
+  describe "Breakfast: Scrabble#score" do
+
+    it "contains a self.score method" do
+      expect(Scrabble::Scrabble).to respond_to :score
+    end
+
+    [
+      [5, "cat"],
+      [12, "kitty"],
+      [10, "rabbit"],
+      [10, "bunny"],
+      [9, "fly"],
+      [9, "unicorn"],
+      [5, "dog"]
+    ].each do |value, word|
+      it "returns #{value} for #{word}" do
+        expect(Scrabble::Scrabble.score(word)).to eq(value)
+      end
+    end
+
+    it "returns 0 for a word longer than 7 letters" do
+      expect(Scrabble::Scrabble.score("supercalifragilisticexpialidocious")).to eq(0)
+    end
+
+  end # describe Breakfast #score...
```

Comment:
I think a spec that shows the score method is case insensitive would be valuable - specs should cover the requirements of the method/class and that was one of the requirements in the read me.